src/transformers/models/prophetnet/modeling_prophetnet.py

```diff
@@ -762,7 +762,7 @@ def forward(
         attn_output = (
             attn_output.view(batch_size, self.num_attn_heads, tgt_len, self.head_dim)
             .transpose(1, 2)
-            .reshape(batch_size, tgt_len, hidden_size)
+            .reshape(batch_size, tgt_len, self.hidden_size)
```

Comment:
The test suite flagged this:
```
AttributeError: 'ProphetNetAttention' object has no attribute 'hidden_size'
```

Reply:
you probably want `config.hidden_size` here.

Reply:
or stash `config.hidden_size` into `self.hidden_size` in init.

Reply:
I'll check again

Reply:
https://github.com/tunib-ai/parallelformers/blob/main/parallelformers/transformers/modeling_prophetnet.py#L29
https://github.com/tunib-ai/parallelformers/blob/main/parallelformers/policies/prophetnet.py#L40

I have modified the model as above before.

https://github.com/tunib-ai/parallelformers/blob/a05780b1d178b4ac5100e42c2b6eec7aedc7dd33/parallelformers/policies/prophetnet.py#L29

It seems that the variable was injected here. We have to change the __init__ method like you said.